packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js

```diff
@@ -35,33 +36,37 @@ const NameType = (props) => {
   return (
     <SpacingsStack scale="xs">
       <PropertyName className="name-type">
-        {isRegex(props.property.name) ? (
-          <Markdown.InlineCode>
-            <span
-              css={css`
-                color: ${designSystem.colors.light.textInfo};
-              `}
-            >
-              /
-            </span>
-            {getExpressionInsideSlashes(props.property.name)[1]}
-            <span
-              css={css`
-                color: ${designSystem.colors.light.textInfo};
-              `}
-            >
-              /
-            </span>
-          </Markdown.InlineCode>
-        ) : (
-          <Markdown.InlineCode>{props.property.name}</Markdown.InlineCode>
-        )}
-        {props.property.required && <Required />}
-        {props.property.beta && (
-          <BetaWrapper>
-            <BetaFlag />
-          </BetaWrapper>
-        )}
+        <SpacingsInline scale="xs">
+          <SpacingsInline scale="xs">
+            {isRegex(props.property.name) ? (
+              <Markdown.InlineCode>
+                <span
+                  css={css`
+                    color: ${designSystem.colors.light.textInfo};
+                  `}
+                >
+                  /
+                </span>
+                {getExpressionInsideSlashes(props.property.name)[1]}
+                <span
+                  css={css`
+                    color: ${designSystem.colors.light.textInfo};
+                  `}
+                >
+                  /
+                </span>
+              </Markdown.InlineCode>
+            ) : (
+              <Markdown.InlineCode>{props.property.name}</Markdown.InlineCode>
+            )}
+            {props.property.required && <Required />}
+          </SpacingsInline>
+          {props.property.beta && (
+            <BetaWrapper>
```

Comment:
Why is the beta flag not part of the inline spacing wrapper but outside?

Reply:
Thanks for catching that, I used an unnecessary extra `SpacingsInline`